.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: perl
+perl:
+    - "5.22"
+    - "5.20"
+    - "5.18"
+    - "5.16"
+    - "5.14"
+
+before_install:
+    - cpanm DBI
+    - cpanm DBD::Pg
+    - cpanm AnyEvent
+    - cpanm POSIX
+    - cpanm Getopt::Long
+    - cpanm Pod::Usage
+    - cpanm Time::HiRes
+
+notifications:
+  email:
```

Comment:
That would have gone to ebow@ebowhalm.com and I don't check that often so wouldn't see it in a timely manner. At least that was the case till I changed the email in the git config to ebow@binary.com.